Cakefile.b

```diff
@@ -0,0 +1,525 @@
+fs                        = require 'fs'
```

Comment:
This is a variant version of the `Cakefile` that contains the task definitions for the `bin/cake` command

Specifically it contains a faster/more encapsulated task for rebuilding when you've made a grammar change:
```
bin/cake build:with-updated-grammar
```

I copied it in from my local Coffeescript repo checkout - there are some ways in which it's diverged from the Coffeescript `Cakefile` but I haven't reconciled them (so I tend to just do `cp Cakefile.b Cakefile` locally to be able to run eg `bin/cake build:with-updated-grammar` but then don't commit this as `Cakefile`)